docs/guidelines-for-setting-up-a-local-workspace.md

```diff
@@ -77,11 +77,12 @@ Under Ubuntu Linux, you can follow the [documentation from the Ubuntu Community]
    --add-exports javafx.controls/com.sun.javafx.scene.control.behavior=com.jfoenix
    --patch-module org.jabref=build/resources/main
    ```
-4. Optional: Use IntellJ to build and run (instead of gradle): File -> Settings -> Build, Execution, Deployment ->  Build Tools -> Gradle -> Under "Build and run using" and "Run tests using" choose "Intellj IDEA"
+4. Use IntellJ to build and run (instead of gradle): File -> Settings -> Build, Execution, Deployment ->  Build Tools -> Gradle -> At "Build and run using" and "Run tests using" choose "Intellj IDEA"
+4. Ensure that JDK13 is enabled for Gradle: Use IntellJ to build and run (instead of gradle): File -> Settings -> Build, Execution, Deployment ->  Build Tools -> Gradle -> Gradle -> Gradle JVM
```

Comment:
"Use IntellJ to build and run (instead of gradle):" can be deleted?!

Reply:
Default installation of IntelliJ has gradle as default. I like that we explain why the settings are as this.